docker/worker/Dockerfile

```diff
@@ -18,13 +18,21 @@ ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update && apt-get upgrade -y && \
     apt-get install -y \
         apt-transport-https \
+        build-essential \
         ca-certificates \
         curl \
         git \
         gnupg-agent \
-        python3-pip \
+        zlib1g-dev \
+        libffi-dev \
+        libssl-dev \
         software-properties-common
 
+RUN curl -sS https://www.python.org/ftp/python/3.11.0/Python-3.11.0.tgz | tar -C /tmp -xzv && \
+    cd /tmp/Python-3.11.0 && \
+    ./configure --enable-optimizations --with-lto --enable-loadable-sqlite-extensions && make && make install && \
```

Comment:
Could you separate the `make`/`make install` to be on separate lines so they're more clear to a reader that this isn't just a `./configure`.